src/webview2/mod.rs

```diff
@@ -176,6 +176,16 @@ impl InnerWebView {
       wparam: WPARAM,
       lparam: LPARAM,
     ) -> LRESULT {
+      if msg == WM_SETFOCUS {
+        // Fix https://github.com/DioxusLabs/dioxus/issues/2900
+        // Get the first child window of the window
```

Comment:
This doesn't sound like the correct behavior for multiwebview windows but i don't reallyyy care about that at the moment and it's still better than losing focus completely.